src/fragments/lib/auth/js/mfa.mdx

```diff
@@ -31,6 +31,7 @@ Auth.setupTOTP(user).then((code) => {
 
 // Then you will have your TOTP account in your TOTP-generating app (like Google Authenticator)
 // Use the generated one-time password to verify the setup
+// This can also be used if user is already signed in for testing/validation
```

Comment:
Didn't notice line 33 when we first spoke about this, maybe some combination of 33 + 34 would be better because 33 already mentions that this method is for verifying the setup, although that's probably a little vague for some users.
I don't think both lines would be needed though